src/methods.js

```diff
@@ -596,7 +596,21 @@ export default {
         named: params => set(params, 'privkeys', map(params.privkeys || [], () => '******'))
       }
     },
-    version: '>=0.7.0'
+    version: '>=0.7.0 <0.18.0'
```

Comment:
We should only add the "deprecation" version when the deprecated method is completely removed, since many deprecated methods are still available via `bitcoind` argument `-deprecatedrpc`. WDYT?

Reply:
I'd also have to make many changes like this as well to my other pull, should we test with -deprecatedrpc set? Otherwise it's a bit confusing.

Reply:
@juscamarena to keep a maximum compatibility to the `bitcoind` RPC API, IMHO we should only block a specific method when it is fully removed.

To answer your question: I think it is not worth the trouble to test against deprecated RPC methods / features, so we shouldn't activate any deprecated RPC on the containers.

Reply:
> We should only add the "deprecation" version when the deprecated method is completely removed, since many deprecated methods are still available via bitcoind argument -deprecatedrpc. WDYT?

From [the release notes for v0.17.0](https://bitcoincore.org/en/releases/0.17.0/):

> `signrawtransaction` is deprecated and will be fully removed in v0.18. 

Since in theory it won't be available at all in version 0.18.0, I thought it made sense to indicate so in the version range.

In version 0.17.0 it is still available with the `deprecatedrpc` option, as you indicated, but that isn't being taken into account here.

Reply:
When investigating the new RPC methods, I've noticed a RPC method in the release notes of a previous version (can't find it right now) that wasn't removed as promised (it remained deprecated). So, I'd prefer to keep it open and then once released define the upper bound and test it against the daemon.

Reply:
Makes sense. I'll update it.